engines/ruffle/assets/run-ruffle.sh

```diff
@@ -8,7 +8,8 @@ set -e
 if [ ! -f run.swf ]; then
     export PATH="$PATH:./jdk-11.0.12/bin/"
     if [[ $LUX_ORIGINAL_EXE_FILE == "Samorost2.exe" ]]; then
-        java -jar ffdec/ffdec.jar -extract "Samorost 2 Original (2005)/Samorost2.exe" -o run.swf biggest
+        cd "Samorost 2 Original (2005)"
+        java -jar "../ffdec/ffdec.jar" -extract "Samorost2.exe" -o run.swf biggest
```

Comment:
Currently, this would run every time, since it's looking for a run.swf in the directory above it. It may make sense to do this check at the top, the cd into the directory, and  then run the ffdec.

Reply:
It outputs "./run-ruffle.sh: line 12: java: command not found"
is java depended on being in a specific folder?